inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/format/DebeziumJsonDynamicSchemaFormat.java

```diff
@@ -44,7 +44,6 @@
  */
 public class DebeziumJsonDynamicSchemaFormat extends JsonDynamicSchemaFormat {
 
-    private static final String IDENTIFIER = "debezium-json";
```

Comment:
Maybe it is better to keep the 'IDENTIFIER'?

Reply:
'IDENTIFIER' is used for distinguish different DynamicSchemaFormat，but 'map<String, ?>'  keys can achieve the same goal. Here if want to keep 'IDENTIFIER', must modify getIdentifier to static function because it is private.